crates/viewer/re_data_ui/src/component_path.rs

```diff
@@ -23,17 +23,17 @@ impl DataUi for ComponentPath {
                 "Indicator component for the {archetype_name} archetype"
             ));
         } else {
-            let results =
-                db.query_caches()
-                    .latest_at(db.store(), query, entity_path, [*component_name]);
+            let results = db
```

Comment:
Get these two at the top of the else block at the very least, and add note about read-coarseness here